Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutRequest.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+@objcMembers
+public class BTPayPalNativeCheckoutRequest: NSObject {
+    public let returnURL: String
+    public let amount: String
+    
+    public init(returnURL: String, amount: String) {
+        self.returnURL = returnURL
+        self.amount = amount
+        super.init()
```

Comment:
Is this `super.init` just a placeholder for now or is it needed for something that I am missing currently?

Reply:
I think we need to call it because we're subclassing `NSObject`.

Reply:
Just checked with Sarah actually we should be good to remove this.